src/components/Cart/CartCount/CartCount.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useState } from 'react';
+import * as S from './CartCountStyle';
+
+const CartCount = () => {
+  const [count, setCount] = useState(1);
+  const handleCount = num => {
+    if (count === 1 && num === -1) return;
+    if (count === 5 && num === 1) return;
+    setCount(count + num);
+  };
+  return (
+    <S.CartCountInput>
+      <S.CartCountMinus
+        onClick={() => {
+          handleCount(-1);
+        }}
+      >
+        -
+      </S.CartCountMinus>
+      <S.CartCountButton>{count}</S.CartCountButton>
+      <S.CartCountPlus
+        onClick={() => {
+          handleCount(+1);
+        }}
+      >
+        +
+      </S.CartCountPlus>
+    </S.CartCountInput>
+  );
+};
```

Comment:
```suggestion
const CartCount = () => {
  const [count, setCount] = useState(1);
  
  const increaseCount = ()=> {
  if(count > 5) return;
    setCount(prev=>prev+1);
  };
  
    const decreaseCount = ()=> {
    if(count <= 1) return;
    setCount(prev=>pre-1);
  };
  
  return (
    <S.CartCountInput>
      <S.CartCountMinus
        onClick={decreaseCount}
      >
        -
      </S.CartCountMinus>
      <S.CartCountButton>{count}</S.CartCountButton>
      <S.CartCountPlus
        onClick={() => {increaseCount}
      >
        +
      </S.CartCountPlus>
    </S.CartCountInput>
  );
};
```

* 이벤트 함수를 분리하고
* 조건에 맞게 return 하고,
* 이전의 state를 참조해서 update하는 함수형 업데이트를 적용하면 
코드가 조금 더 읽고 쓰기 편해집니다.

Reply:
네 알겠습니다